spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -493,13 +493,14 @@ Alternatively, a scope type may be specified using a stereotype annotation, as d
 
 ==== Default scope
 
-When no scope is explicitly declared by annotating the bean class or producer method or field the scope of a bean is defaulted.
+When no scope is explicitly declared by annotating the bean class or producer method or field and the bean is not deployed in a <<trimmed_bean_archive>> the scope of a bean is defaulted.
```

Comment:
Yes there are 2 ways to word what we aim for. Either declare that it doesn't get a default scope and thus is not a Bean. Or to veto it. Both ways lead to the same behaviour and are internally pretty much the same